src/InteractsWithBus.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Zenstruck\Messenger\Test;
+
+use Zenstruck\Messenger\Test\Bus\TestBus;
+
+trait InteractsWithBus
```

Comment:
I'm thinking we should merge this with `InterfactsWithMessenger`: as a developer I would want to only add one trait to use the potential of this library. WDYT @kbond ?

Reply:
My mistake about this one...

Too many times with my main project and forgot how it was built here.

Reply:
no problem :smile: